packages/vitest/src/defaults.ts

```diff
@@ -23,6 +22,7 @@ const defaultCoverageExcludes = [
   '**/__tests__/**',
   '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress}.config.{js,cjs,mjs,ts}',
   '**/.{eslint,mocha,prettier}rc.{js,cjs,yml}',
+  '{vite,vitest}.config.*',
```

Comment:
No, what I meant is to also put files from `:23` line in `defaultExclude` 😄 

23 line already has vite and vitest configs :)